src/main/java/address/controller/StatusBarFooterController.java

```diff
@@ -136,6 +136,16 @@ public void handleUpdaterCompletedEvent(UpdaterFinishedEvent ufe) {
     }
 
     @Subscribe
+    public void handleUpdaterFailedEvent(UpdaterFailedEvent ufe) {
+        Platform.runLater(() -> {
+            updaterStatusBar.setText(ufe.toString());
+            updaterStatusBar.setProgress(0.0);
+            updaterStatusBar.setText("");
+            saveLocationLabel.setVisible(true);
```

Comment:
I am following the original implementation of the `StatusBarFooterController`. I think the intention was not to show `saveLocationLabel` if updater is working, and only show it once updater is done. Hence the save location here.